src/Confluent.Kafka/Producer.cs

```diff
@@ -1126,16 +1144,22 @@ public void ProduceAsync(string topic, TKey key, TValue val, bool blockIfQueueFu
         ///     Raised when there is information that should be logged.
         /// </summary>
         /// <remarks>
-        ///     You can specify the log level with the 'log_level' configuration
-        ///     property. You also probably want to specify one or more debug
-        ///     contexts using the 'debug' configuration property.
+        ///     Note: By default not many log messages are generated.
```

Comment:
dito.

Also, there is really no point in setting the log_level, you could probably just remove any reference to it.
If you enable debugging the log_level is automatically increased to 7.
But none of this is very interesting to app developers, and the info is available in CONFIGURATION.md.
There are more important concepts to explain than this one.